lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -823,7 +823,7 @@ def _on_get_password_action(self, event: ActionEvent):
 
         password = self.secrets.get(Scope.APP, self.secrets.password_key(user_name))
         cert = self.secrets.get_object(
-            Scope.APP, CertType.APP_ADMIN.val
+            Scope.APP, CertType.APP_ADMIN.val, peek=True
```

Comment:
I understand why we're doing this - but I'm a bit worried about the full synchronisation issues this may entail cross units.
What broader impacts are we expecting to have here?